### ISSUE TITLE
How can I delete my account?

### ISSUE THREAD

**Issue by cloneofghosts** [Pirate-Weather/pirateweather#192]
### Discussed in https://github.com/Pirate-Weather/pirateweather/discussions/182

<div type='discussions-op-text'>

<sup>Originally posted by @betapictoris April  2, 2024</sup>
I've searched through the web interface, but I can't seem to find a place to delete my account.  I think it's an amazing API, but I simply don't use it anymore, how can I delete my account?</div>

Email has been sent to [api@alexanderrey.ca](mailto:api@alexanderrey.ca) and will also tag @alexander0042 to look into this issue.

**Comment by cloneofghosts** [comment]
@betapictoris You get this solved at all? If not @alexander0042 are you able to look into this?

**Comment by alexander0042** [comment]
Yup! Replied via e-mail, so should be solved! For anyone else looking, account deletion in on Apiable's radar, but for now just shoot me an e-mail and I'll delete it manually.
